src/locales/en.json

```diff
@@ -23,6 +23,7 @@
     "menu_upload": "Upload files",
     "menu_new_folder": "New folder",
     "menu_select": "Select items",
+    "menu_download_all": "Download folder",
```

Comment:
The key and the value are not saying the same. It will be confusing later if it is not yet.